README.md

```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
 
 For example, if you add the contents of the `dist` folder to the `assets/office-js` directory of your project, then you'd add the following `<script>` tag to your HTML file:
 
-    <script src="/assets/office-js/office.js"></script>
-
+```html
+<script src="/assets/office-js/office.js"></script>
+```
 <br />
 
 ## IntelliSense definitions
 
 TypeScript definitions for Office.js are available.
 
-* For latest **RELEASE** version of Office.js:
-  * DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js/index.d.ts>
-  * @types: `@types/office-js`.  (Acquire as `npm install @types/office-js --save-dev`)
-
-* For **any version** (including **RELEASE**, but also including **BETA**, etc.):
-  * Inside of the NPM package, under `dist/office.d.ts`
-  * In this repo: [dist/office.d.ts](dist/office.d.ts)
+- For latest **RELEASE** version of Office.js:
+  - DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js/index.d.ts>
+  - @types: `@types/office-js`.  (Acquire as `npm install @types/office-js --save-dev`)
+- For latest **BETA** version of Office.js:
+  - DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js-preview/index.d.ts>
+  - @types: `@types/office-js-preview`.  (Acquire as `npm install @types/office-js-preview --save-dev`)
+- For **any** version:
+  - Inside of the NPM package, under `dist/office.d.ts`
+  - In this repo: [dist/office.d.ts](dist/office.d.ts)
 
 ### Using TypeScript definitions with the NPM package
```

Comment:
We should probably fix the use of gerund verbs in headings at some point!

Reply:
fixed